cmd/kubelet/app/server.go

```diff
@@ -160,8 +163,6 @@ func (s *KubeletServer) Run(_ []string) error {
 		glog.Warningf("No API client: %v", err)
 	}
 
-	glog.Infof("Using root directory: %v", s.RootDirectory)
```

Comment:
@thockin Heh, I had left this in from an earlier secrets PR and was annoyed at myself for letting my debug stuff slip in.  I'll add it back.